doc/Language/nativecall.pod6

```diff
@@ -653,9 +653,18 @@ say $esponja;
 
 NativeCall also supports native functions that take functions as arguments.  One
 example of this is using function pointers as callbacks in an event-driven
-system.  When binding these functions via NativeCall, one need only provide the
+system.  When binding these functions via NativeCall, one should need only provide the
```

Comment:
need only provide is correct, should need only is not. *should need to only provide* could be an alternative.

Reply:
needs only provide, actually...